src/binding/petsc4py/src/petsc4py/PETSc/DMComposite.pyx

```diff
@@ -61,7 +142,23 @@ cdef class DMComposite(DM):
             clvecs[i] = (<Vec?>lvecs[<Py_ssize_t>i]).vec
         CHKERR( DMCompositeGatherArray(self.dm, cimode, gvec.vec, clvecs) )
 
-    def getGlobalISs(self):
+    def getGlobalISs(self) -> list[IS]:
```

Comment:
another strange thing in the API. getEntries returns a tuple, this returns a list. May add a FIXME also here?